src/operator/controllers/intents_reconcilers/otterizecloud/cloud_api.go

```diff
@@ -38,3 +39,14 @@ func (c *CloudApiImpl) ReportKubernetesNamespace(ctx context.Context, namespace
 	}
 	return nil
 }
+
+func (c *CloudApiImpl) ReportKafkaServerConfig(ctx context.Context, namespace string, source string, server graphqlclient.KafkaServerConfigInput) error {
+	res, err := graphqlclient.ReportKafkaServerConfig(ctx, c.client, namespace, source, server)
+	if err != nil {
+		return err
+	}
+	if res.ReportKafkaServerConfig {
```

Comment:
What happens if it's false? Function didn't succeed but returns no error? Even though GQL returns a value, since only the error matters - please ignore the return value and only use the error.

Reply:
Well, the way I wrote it is a mistake, but actually if `err == nil` so the boolean is true. 
We should definitely find a way to return Null